source/framework/vx_graph_verify.c

```diff
@@ -964,7 +1033,7 @@ static vx_status ownGraphCalcHeadAndLeafNodes(vx_graph graph)
     return status;
 }
 
-static vx_status ownGraphAllocateDataObject(vx_node node_cur, uint32_t prm_cur_idx, vx_reference ref)
+vx_status ownGraphAllocateDataObject(vx_graph graph, vx_node node_cur, uint32_t prm_cur_idx, vx_reference ref)
```

Comment:
the parameter "graph" is not used in this function and should be removed (according to MISRA). This will probably mean it should be removed from the parameter list of some of the functions that call this, and so on...